Classes/Twig/Typo3Loader.php

```diff
@@ -36,7 +38,11 @@ public function getSourceContext($name)
     {
         $path = $this->findTemplate($name);
```

Comment:
Actually this should throw an exception already. The problem is in this line:

```
private function findTemplate(string $name, bool $throw = false): string
```

`$throw` should be true. I "copied" the behavior from `\Twig_Loader_Filesystem::findTemplate` because it is very similar. Looks like I placed `false` instead of `true` there.

The mentioned line should become this:

```
private function findTemplate(string $name, bool $throw = true): string
```